index.js

```diff
@@ -172,19 +176,22 @@ client.once('disconnect', () => { console.log('Status: Disconnected'); botState
 
 client.on('message', async message => {
 
+    
+
     console.log('\n\n\nLog: Message: ' + message.content);
-    console.log("Log: First argument: " + message.content.split(" ")[0] + " ," + message.content.split(" ")[0].length);
+    let cmd = message.content.split(" ")[0];

```

Comment:
would prefer if debug messages stay tgt; please move the declaration to above or below the debug messages